src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -299,6 +301,10 @@ export class MeetingManager implements AudioVideoObserver {
     if (videoDownlinkBandwidthPolicy) {
       configuration.videoDownlinkBandwidthPolicy = videoDownlinkBandwidthPolicy;
     }
+    // TODO: Combine the meeting manager config and meeting session config 
```

Comment:
This TODO does not make sense. We cannot combine meeting manager config and meeting session config. 

Meeting manager config include properties like `activeSpeakerPolicy`, `simulcastEnabled` and `enbaleWebAudio` which are not part of meeting session config but are part of meeting manager config.

Reply:
Well, I think what Michael means is meetingSessionConfiguration can be a param inside the meetingManagerConfig which is valid right?

Reply:
Also, we can remove the TODO since we do not yet have any "meeting session config".